src/com/sun/ts/tests/signaturetest/signature-repository/javax.ejb.sig_3.2_se8

```diff
@@ -135,110 +135,110 @@ meth public abstract boolean isDone()
 meth public abstract {java.util.concurrent.Future%0} get() throws java.lang.InterruptedException,java.util.concurrent.ExecutionException
 meth public abstract {java.util.concurrent.Future%0} get(long,java.util.concurrent.TimeUnit) throws java.lang.InterruptedException,java.util.concurrent.ExecutionException,java.util.concurrent.TimeoutException
 
-CLSS public javax.ejb.AccessLocalException
+CLSS public jakarta.ejb.AccessLocalException
```

Comment:
Please revert change to this file, the *.se8 signature files will be automatically generated as described on [signaturetest/README](https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/src/com/sun/ts/tests/signaturetest/README).

Reply:
Thanks for the review Scott, I reverted this on b0ce81f